README.md

```diff
@@ -102,6 +102,15 @@ Operator API will be available at [http://127.0.0.1:8000/operator/v1/](http://12
 
 Internal API will be available at [http://127.0.0.1:8000/internal/v1/](http://127.0.0.1:8000/internal/v1/)
 
+### Generating API documentation
+
+The API documentation conforms to [Swagger Specification 2.0](http://swagger.io/specification/). Use
+[swagger-codegen](https://github.com/swagger-api/swagger-codegen) to generate the documentation.
+
+For example, to generate static HTML documentation for operator APIs:
```

Comment:
I would add some more info, for example something as follows. I love verbose instructions ;)

    You need Maven to build the Swagged-Codegen client:

        # Ubuntu
        sudo apt-get install maven

    Due to [a bug in Codegen](https://github.com/swagger-api/swagger-codegen/pull/4508), we're using an unreleased version at the moment. Clone Codegen master branch and build it.

        git clone https://github.com/swagger-api/swagger-codegen 
        cd swagger-codegen/
        mvn clean package  # Takes a few minutes

    The client will now be available at `modules/swagger-codegen-cli/target/swagger-codegen-cli.jar`.

    To build the docs, in `parkkihubi` repository root:

        cd docs/api
        java -jar /path/to/codegen/swagger-codegen-cli.jar generate -i internal.yaml -l html2 -c config.json -o /output/path/internal/
        java -jar /path/to/codegen/swagger-codegen-cli.jar generate -i operator.yaml -l html2 -c config.json -o /output/path/operator/